test-integration/helpers/wait.coffee

```diff
@@ -0,0 +1,67 @@
+selenium = require 'selenium-webdriver'
+toLocator = require './to-locator'
+windowPosition = require './window-position'
+
+class Wait
+  constructor: (test) -> @test = test
+
+  forMultiple: (locator, ms = 60 * 1000) ->
+    locator = toLocator(locator)
+    start = null
+    timeout = 0
+    @test.driver.call => # Enqueue the timeout to increase only once this starts
+      start = Date.now()
+      @test.addTimeoutMs(ms)
+    @test.driver.wait(selenium.until.elementsLocated(locator))
+    .then (val) =>
+      end = Date.now()
+      spent = end - start
+      diff = ms - spent
+      # console.log "Took #{spent / 1000}sec of #{ms / 1000}"
+      if spent > ms
+        throw new Error("BUG: Took longer than expected (#{spent / 1000}). Expected #{ms / 1000} sec")
+      @test.addTimeoutMs(-diff)
+      val
+    # Because of animations an element might be in the DOM but not visible
+    el = @test.driver.findElements(locator)
+
+    el.then (elements) =>
+      @test.driver.wait(selenium.until.elementIsVisible(elements[0]))
```

Comment:
I dunno.  I just copied the codez in :)  Actually why are we waiting until they're visible anyway?  Do we really need to?  I could see that the first element might be visible but other's might be below the scroll.